.github/workflows/check.yml

```
@@ -148,7 +148,7 @@ jobs:
           # respective default features only. Can reveal warnings otherwise
           # hidden given that a plain cargo clippy combines all features of the
           # workspace. See e.g. https://github.com/mozilla/neqo/pull/1695.
-          cargo +${{ matrix.rust-toolchain }} hack clippy --all-targets -- -D warnings || ${{ matrix.rust-toolchain == 'nightly' }}
+          cargo +${{ matrix.rust-toolchain }} hack clippy --all-targets --each-feature --exclude-features gecko -- -D warnings || ${{ matrix.rust-toolchain == 'nightly' }}
```

```suggestion
          cargo +${{ matrix.rust-toolchain }} hack clippy --all-targets --feature-powerset --exclude-features gecko -- -D warnings || ${{ matrix.rust-toolchain == 'nightly' }}
```

In case it doesn't add to much delay, how about the powerset instead of each feature individually?

https://github.com/taiki-e/cargo-hack?tab=readme-ov-file#--feature-powerset

---

Let's try.